src/modules/auth/webauthn/WebAuthnScreen.js

```diff
@@ -181,6 +197,25 @@ function WebAuthnCard(props) {
 		}
 	}
 
+	// Edit authenticator name
+	const onSubmitKeyName = async (values) => {
+		let response;
+		try {
+			response = await SeaCatAuthAPI.put(`/public/webauthn/${values.id}`, {"name": `${values.name}`});
+			// TODO: enable validation, when ready in SA service
```

Comment:
@Aringoaway Is this TODO still valid? Since you have a validation on auhenticator name, right?